contracts/src/core/IAlignedLayerServiceManager.sol

```diff
@@ -26,6 +26,7 @@ interface IAlignedLayerServiceManager {
         uint256 available
     ); // 5c54305e
     error InvalidQuorumThreshold(uint256 signedStake, uint256 requiredStake); // a61eb88a
+    error WithdrawalFailed(address batcher, uint256 amount); // 92873d13
```

Comment:
it seems like this error is not being used

Reply:
Solved in [0a20dc5](https://github.com/yetanotherco/aligned_layer/pull/872/commits/0a20dc534da08da59dc055c0770b109c62d6141d)